libraries/AP_GPS/AP_GPS.h

```diff
@@ -627,6 +627,13 @@ class AP_GPS
         GPS_AUTO_CONFIG_ENABLE  = 1
     };
 
+    enum GPS_AUTO_SWITCH {
```

Comment:
Better use an enum class

Reply:
agreed, I'll change it to an enum class.

Reply:
Still waiting on this :-)

Reply:
@magicrub ping

Reply:
ok ok!

Reply:
changed to enum class GPS_AUTO_SWITCH in a force push